org.flatpak.Builder.BaseApp.json

```diff
@@ -103,7 +153,7 @@
             "sources": [
                 {
                     "type": "file",
-                    "url": "https://gitlab.archlinux.org/archlinux/packaging/packages/rpmextract/-/raw/main/rpm2cpio",
+                    "url": "https://gitlab.archlinux.org/archlinux/packaging/packages/rpmextract/-/raw/9763231c8ae0539c6d7fe8ad31967b2175089743/rpm2cpio",
```

Comment:
This hasn't been updated in 15 years and is broken... it's outputting binary data on the command line.

Will replace with `rpm2cpio` from `rpm` which Fedora uses by default.

Reply:
https://github.com/flathub/org.flatpak.Builder.BaseApp/actions/runs/9730265418/job/26853137441?pr=25#step:14:4